nsis/pcsx2_full_install.nsi

```diff
@@ -45,6 +53,69 @@ ShowUninstDetails nevershow
 ; =======================================================================
 
 ; -----------------------------------------------------------------------
+
+; The "" makes the section hidden.
+Section "" SecUninstallPrevious
+
+	Call UninstallPrevious
+	
+SectionEnd
+
+Function UninstallPrevious
+	;Check for uninstaller.
+	ReadRegStr $R0 HKLM "${INSTDIR_REG_KEY}" "UninstallString"
+	${If} $R0 == ""
+		Goto Done
+	${EndIf}
+	
+	;Check if any other version is installed
+	ReadRegStr $R1 HKLM "${INSTDIR_REG_KEY}" "InstalledVersion"
+	
+	;This check for older versions (pre 1.6.0) without InstalledVersion string will bypass this section
+	${If} $R1 == ""
+	DetailPrint "InstalledVersion string not found, skipping version check"
```

Comment:
Can't it be assumed that if `InstalledVersion` is not in the registry, then the older version is installed? This installer would be used only for the newer builds, wouldn't it?